src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Inputs/BitChoiceGroupTests.cs

```diff
@@ -7,66 +7,196 @@ namespace Bit.Client.Web.BlazorUI.Tests.Inputs
+                parameters =>
+                {
+                    parameters.Add(p => p.Text, text);
+                });
+
+            var bitChoiceOptionLabelText = component.Find(".bit-cho label span");
+
+            Assert.AreEqual(text, bitChoiceOptionLabelText.TextContent);
+        }
+
+        [DataTestMethod,
+          DataRow(true),
+          DataRow(false)
+        ]
+        public void BitChoiceOptionShouldAcceptGivenCheckStatus(bool isChecked)
+        {
+            var component = RenderComponent<BitChoiceGroupTest>(
+                parameters =>
+                {
+                    parameters.Add(p => p.IsChecked, isChecked);
+                });
+
+            var input = component.Find(".bit-cho input");
+
+            Assert.AreEqual(isChecked, input.HasAttribute("checked"));
+
+        }
+
+        [DataTestMethod,
+           DataRow(true, true),
+           DataRow(true, false),
+           DataRow(false, true),
+           DataRow(false, false),
+        ]
+        public void BitChoiceOptionMustRespondToTheClickEvent(bool groupIsEnabled, bool optionIsEnabled)
+        {
+            var component = RenderComponent<BitChoiceGroupTest>(
+                parameters =>
+                {
+                    parameters.Add(p => p.ChoiceGroupIsEnabled, groupIsEnabled);
+                    parameters.Add(p => p.ChoiceOptionIsEnabled, optionIsEnabled);
+                });
+
+            var bitChoiceOptionInput = component.Find(".bit-cho input");
+
+            bitChoiceOptionInput.Click();
+
+            if (groupIsEnabled && optionIsEnabled)
+            {
+                //Assert.IsTrue(bitChoiceOptionInput.HasAttribute("checked"));
+            }
+
+            //Assert.AreEqual(groupIsEnabled && optionIsEnabled ? 1 : 0, component.Instance.ChoiceOptionClickedValue);
+        }
+
+        [DataTestMethod,
+          DataRow(true, true, 2),
+           DataRow(true, false, 2),
+           DataRow(false, true, 2),
+           DataRow(false, false, 2),
+       ]
+        public void BitChoiceOptionMustRespondToTheChangeEvent(bool groupIsEnabled, bool optionIsEnabled, int count)
+        {
+            var component = RenderComponent<BitChoiceGroupTest>(
+                parameters =>
+                {
+                    parameters.Add(p => p.ChoiceGroupIsEnabled, groupIsEnabled);
+                    parameters.Add(p => p.ChoiceOptionIsEnabled, optionIsEnabled);
+                });
+
+            var input = component.Find(".bit-cho input");
+
+            input.Click();
+
+            if (groupIsEnabled && optionIsEnabled)
+            {
+                //Assert.IsTrue(input.HasAttribute("checked"));
+            }
+
+            //Assert.AreEqual(groupIsEnabled && optionIsEnabled ? count.ToString() : null, component.Instance.Value);
+            //Assert.AreEqual(groupIsEnabled && optionIsEnabled ? 1 : 0, component.Instance.ChoiceOptionChangedValue);
```

Comment:
please add a TODO comment to explain why these lines are commented, just like the ones commented before because of issues in the Bunit framework.